portal/apps/org.wso2.carbon.iam.admin-portal.app/src/main/pages/users/list.js

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+function onGet(env) {
+    var session = getSession();
+    var usernameClaim = getUsernameClaimFromProfile();
+    var userList = getUsersForList(0, 10, "", usernameClaim);
+    return {userList: userList};
+}
+
+
+function onPost(env) {
+
+}
+
+function getUsersForList(offset, length, domainName, usernameClaim) {
+    var userList;
+    try {
+        userList = callOSGiService("org.wso2.is.portal.user.client.api.UserMgtClientService",
+            "getUsersForList", [offset, length, domainName, usernameClaim]);
+    } catch (e) {
+        return {errorMessage: 'signup.error.retrieve.domain'};
+    }
+    return userList;
+}
+
+function getDomainNames(env) {
+    var domainNames;
+    if (env.config.isDomainInLogin) {
```

Comment:
Can you please explain the purpose of this config 'isDomainInLogin'? What is the behavior if this is set to false?